autoload/neomake/makers/ft/rust.vim

```diff
@@ -22,3 +22,21 @@ function! neomake#makers#ft#rust#rustc() abort
             \ '%-G%s',
         \ }
 endfunction
+
+function! neomake#makers#ft#rust#cargo() abort
+    return {
+        \ 'args': ['build'],
+        \ 'append_file': 0,
+        \ 'errorformat':
+            \ ',' .
+            \ '%-G,' .
+            \ '%-Gerror: aborting %.%#,' .
+            \ '%-Gerror: Could not compile %.%#,' .
+            \ '%Eerror: %m,' .
+            \ '%Eerror[E%n]: %m,' .
+            \ '%-Gwarning: the option `Z` is unstable %.%#,' .
+            \ '%Wwarning: %m,' .
+            \ '%Inote: %m,' .
+            \ '%C %#--> %f:%l:%c',
+        \ }
+endfunction
```

Comment:
Shouldn't this be built on top of, or at least be more similar to `neomake#makers#cargo#cargo`?
https://github.com/neomake/neomake/blob/eceb9fafa41190e7c7b270fbfd13d299891f5b4a/autoload/neomake/makers/cargo.vim#L3-L9